### ISSUE TITLE
Refactor `PTO` to `order` as in EKO?

### ISSUE THREAD

**Issue by felixhekhorn** [N3PDF/yadism#151]
- @giacomomagni mirrored in #145 the theory card refactor from eko and introduced the `order` field
- I wonder whether instead we should call `eko.compatibility` inside and then act further (note that we need the eko settings for `Couplings.from_dict`
- as far as yadism is concerned the second element of `order` is never relevant (since QED corrections are out of scope, as discussed), instead the tuple `(PTODIS, PTO)` is relevant

**Comment by felixhekhorn** [comment]
We need some compatibility between eko and yadism, since they share some settings and, more relevant, some code: `Couplings` (which explicitly depends on QED) - that's why I suggested to build on top of `eko.compatibility`

**Comment by AleCandido** [comment]
> We need some compatibility between eko and yadism, since they share some settings and, more relevant, some code: `Couplings` (which explicitly depends on QED) - that's why I suggested to build on top of `eko.compatibility`

Not that sure: I'd like runcards to be independent, because there are a lot of theory parameters that are DIS only and Evolution only, and they should not enter each other cards (since we are not sensitive to it, they should not be provided).
Moreover, `yadism` is performing a different job than `eko`, the dependency is just an implementation detail (to save us some duplication), but should be kept internal: from the point of view of the user, they are separate and that's it.

Also consider that `compatibility` is a temporary layer, going to vanish as soon as possible (if we need compatibility with database, it should be dealt at the database interface level - i.e. in `pineko`).

**Comment by giacomomagni** [comment]
I'm happy with this solution: 
```
# EKO
order: ["QCD", "QED"]
# yadism
order: "QCD"
evolution_order:`["QCD", "QED"]`
```
apart that `"QCD"` values will not have the same meaning, since we have a +1 for eko ...

**Comment by AleCandido** [comment]
> apart that `"QCD"` values will not have the same meaning, since we have a +1 for eko ...

Yes, you're right wrt the pre-QED status. But after QED modification, they should be the exact same, i.e. the number of powers of $\frac{\alpha_s}{4\pi}$ (since we needed also the case with no QCD, and pure QED, i.e. `[0, 1]`).
